### ISSUE TITLE
Checked checkboxes in multi-select dropdown do not honor high contrast

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#4113]
Windows 10 and Edge.
The checked checkboxes still use themePrimary in the dropdowns. The actual checkbox control is ok.
![image](https://user-images.githubusercontent.com/20363037/36741418-ebc41e88-1b99-11e8-9cda-fa8b61a52c76.png)

ِActually the checkbox control isn't right either.
![image](https://user-images.githubusercontent.com/20363037/36741808-e88e2726-1b9a-11e8-9220-83a769b59eeb.png)

**Comment by jordandrako** [comment]
@betrue-final-final Would the proper way to change the color be:

```js
{
  selectors: {
    [HighContrastSelector]: {
      background: 'white',
    }
  }
}
```

or something along these lines:

```js
{
  selectors: {
    [HighContrastSelector]: {
      background: 'windowtext',
    }
  }
}
```

I'm thinking probably the latter as only the window knows what kind of high contrast mode, but I don't know what color name to use (windowtext give that yellow color on HCM1).

**Comment by betrue-final-final** [comment]
You are correct. Windowtext would be best.

**Comment by betrue-final-final** [comment]
What is HCM1?

**Comment by jordandrako** [comment]
High Contrast Mode 1. I also found a list of CSS2 color codes, including "HightlightText" which ends up being white in this case where "WindowText" is yellow. However, trying to change the Icon color to "Window" results in it turning into a black rectangle.

**Comment by lynamemi** [comment]
This one can be closed, right @jordandrako ?

**Comment by jordandrako** [comment]
Yeah, weird it didn't auto close.
